themes/mongodb/static/src/css/tabs.css

```diff
@@ -0,0 +1,79 @@
+/**********************************************/
+/*Display and hide the appropriate tab content*/
+.tab {
+    display:none;
+}
+
+.tab-active {
+    display:block;
+}
+
+.nav-tabs li {
+    background-color: #f4f6f6;
+}
+/**********************************************/
```

Comment:
Was trying to make it super obvious which sections had which css in it. Doesn't need to be here, removed.